test/table.template

```diff
@@ -0,0 +1,76 @@
+  CREATE TABLE "?"."LOCALUSER1"
+   (    "ID" VARCHAR(20) NOT NULL,
```

Comment:
I just took it as it was on the original [tables.sql](https://github.com/strongloop/loopback-connector-db2/blob/master/test/tables.sql) file. So I assume it is needed.